recipe/meta.yaml

```diff
@@ -1,13 +1,17 @@
+{% set version = "1.2.3" %}
+
 package:
   name: artview
-  version: "1.2"
+  version: {{ version }}
 
 source:
-  git_url: https://github.com/nguy/artview
-  git_rev: v1.2
+  fn: artview-{{ version }}.tar.gz
+  url: https://github.com/nguy/artview/releases/download/Release_v1.2/artview-{{ version }}.tar.gz
```

Comment:
Could we change this to `https://github.com/nguy/artview/archive/v{{ version }}.tar.gz`?